docs/tr/week08/08-1.md

```diff
@@ -0,0 +1,204 @@
+
+## Diger Karşıtsal Yontemler 
+
+
+Karşıtsal ıraksama(Contrastive divergence), Oran Eşleştirme(Ratio Matching) , Gürültüye Karşıt Kestirim(Noise Contrastive Estimation), ve Minimum OLasılık Akışı(Minimum Probability Flow) gibi çeşitli karşıtsal yöntemler lıteratürde bulunmaktadır. Bu bölümde, karşıtsal ıraksamanın arkasında yatan temel fikirden bahsedeceğiz. 
+
+<!-- ## Other Contrastive Methods
+There are other contrastive methods such as contrastive divergence, Ratio Matching, Noise Contrastive Estimation, and Minimum Probability Flow. We will briefly discuss the basic idea of contrastive divergence. -->
+
+### Karşıtsal Iraksama(Contrastive Divergence)
+
+Karşıtsal Iraksama(CD) girdileri akkılıca bozarak veri ggösterimi öğrenmeyi hedefler. 
+Sürekli bir uzayda, önce eğitim kümemizden bir örnek $y$ seçip  sonra onun enerjisinin düşürüyoruz. Seçtiğimiz örnek için, gradyan bazlı bir süreç ile gürültü kullanarak enerji yüzeyinde hareket ediyoruz. Eğer girdilerimiz ayrık bir uzaydan alınıyorsa, örneği rastgele bozmamız o örneğin enerjisini değiştirmek için yeterli olucaktır. Bozunum sonucu eğer enerji düşüyor ise o örneği tutuyoruz. Aksi takdirde, belli bir olaslılık ile o örneği atıyoruz. Bu sürecin devam ettirdikçe $y$'nin enerjisi de en sonunda düşecektir. Daha sonra, enerji fonksiyonumuzun parametresini $y$ ve zıtlaştırılmış örneğimiz $\bar y$'i kayıp fonksiyonu kullanarak kıyaslayıp güncelleyebilmiş oluruz.
+
+<!-- ### Contrastive Divergence
+Contrastive divergence is another model that learns the representation by smartly corrupting the input sample. In a continuous space, we first pick a training sample $y$ and lower its energy. For that sample, we use some sort of gradient-based process to move down on the energy surface with noise. If the input space is discrete, we can instead perturb the training sample randomly to modify the energy. If the energy we get is lower, we keep it. Otherwise, we discard it with some probability. Keep doing so will eventually lower the energy of $y$. We can then update the parameter of our energy function by comparing $y$ and the contrasted sample $\bar y$ with some loss function. -->
+
+
+### Kalıcı Karşıtsal Iraksama(Persistent Contrastive Divergence)
+
+Karşıtsal Iraksama'nın özel bir versiyonu ise Kalıcı Karşıtsal Iraksama yöntemidir. Önceki yönteme kıyasla sistem, konumunu hatırlayabilen bir grup  "parçacık" kullanır. Bu parçacıklar tıpkı CD'daki gibi enerji yüzeyinde hareket ettirilir. Sonunda, bu parçaçıklar enerji yüzeyimizdeki düşük enerjili yerleri bulucak ve yukarı itilmelerni sağlayacaktır. Ancak, boyut arttıkça sistem performansı kötü etkilenmektedir.
```

Comment:
```suggestion
Karşıtsal Iraksama'nın özel bir versiyonu ise Kalıcı Karşıtsal Iraksama yöntemidir. Önceki yönteme kıyasla sistem, konumunu hatırlayabilen bir grup "parçacık" kullanır. Bu parçacıklar tıpkı CD'deki gibi enerji yüzeyinde hareket ettirilir. Sonunda, bu parçaçıklar enerji yüzeyimizdeki düşük enerjili yerleri bulacak ve yukarı itilmelerini sağlayacaktır. Ancak, boyut arttıkça sistem performansı kötü etkilenmektedir.
```

Reply:
daki is ok because in Turkish you say it for karşıtsal ıraksama'daki. Even for English-like Turkish, it's contrastive divergence'daki (I know last example is not valid for Turkish). Resolving for others, but please fix 'daki for review PR.

Reply:
http://tdk.gov.tr/icerik/yazim-kurallari/kisaltmalar/ , relevant part;

> Küçük harflerle yapılan kısaltmalara getirilen eklerde kelimenin okunuşu esas alınır: cm’yi, kg’dan, mm’den, kr.un. Büyük harflerle yapılan kısaltmalara getirilen eklerde ise kısalt­manın son harfinin okunuşu esas alınır: BDT’ye, TDK’den, THY’de, TRT’den, TL’nin vb. Ancak kısaltması büyük harflerle yapıldığı hâlde bir kelime gibi okunan kısaltmalara getirilen eklerde kısaltmanın okunuşu esas alınır: ASELSAN’da, BOTAŞ’ın, NATO’dan, UNESCO’ya vb.